plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1098,9 +1195,8 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    void resume() {
       _pause_production = false;
       // reset vote received so production can be explicitly resumed, will pause again when received vote time limit hit again
-      auto now = fc::time_point::now();
-      _last_other_vote_received.store(now, std::memory_order_relaxed);
-      _last_producer_vote_received.store(now, std::memory_order_relaxed);
+      if (_is_savanna_active)
+         _implicit_pause_vote_tracker.activate();
```

Comment:
Here you would instead make the call:

```cpp
_implicit_pause_vote_tracker.force_unpause();
```

You may still wish to guard it with `if(_is_savanna_active)` as an optimization, but it would be safe to call it regardless of whether Savanna was active or not.